dev/com.ibm.websphere.appserver.features/visibility/public/jsf-2.2/com.ibm.websphere.appserver.jsf-2.2.feature

```diff
@@ -23,14 +23,17 @@ IBM-API-Package: javax.faces; type="spec", \
  javax.faces.validator; type="spec", \
  javax.faces.view; type="spec", \
  javax.faces.view.facelets; type="spec", \
- javax.faces.webapp; type="spec"
+ javax.faces.webapp; type="spec", \
+ org.apache.myfaces; type="third-party", \
+ org.apache.myfaces.*; type="third-party"
```

Comment:
Do we need to export org.apache.myfaces and myfaces.*?

Reply:
Good catch - there aren't any classes in the `org.apache.myfaces` package, we only need `org.apache.myfaces.*`